data/json/recipes/recipe_deconstruction.json

```diff
@@ -125,7 +125,93 @@
     {"id":"HAMMER","level":1,"amount":1}
   ], "components": [
     [
-      [ "copper", 70 ]
+      [ "copper", 173 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "spear_copper",
+  "id_suffix": "uncraft",
+  "category": "CC_NONCRAFT",
+  "subcategory": "CSC_NONCRAFT",
+  "difficulty": 0,
+  "time": 1200,
+  "reversible": true,
+  "autolearn": false,
+  "qualities":[{"id":"HAMMER","level":1,"amount":1}],
+  "components": [
+    [ [ "stick", 1 ] ],
+    [ [ "copper", 325 ] ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "copper_knife",
+  "id_suffix": "uncraft",
+  "category": "CC_NONCRAFT",
+  "subcategory": "CSC_NONCRAFT",
+  "difficulty": 0,
+  "time": 600,
+  "reversible": true,
+  "autolearn": false,
+  "qualities":[
+    {"id":"HAMMER","level":1,"amount":1}
+  ], "components": [
+    [
```

Comment:
Oh wait, those are just uncrafts.

No need to have separate uncrafts when you have crafting recipes for this stuff.